### ISSUE TITLE
POSIX::exit is the same as builtin exit [rt.cpan.org #42885]

### ISSUE THREAD

**Issue by toddr** [toddr/IPC-Run#78]
Migrated from [rt.cpan.org#42885](https://rt.cpan.org/Ticket/Display.html?id=42885) (status was 'open')

Requestors:
* GAAS@cpan.org

From gaas@cpan.org on 2009-01-29 12:25:00:
```
I see code like this in IPC::Run:

   ## Use POSIX::exit to avoid global destruction, since this might
   ## cause DESTROY() to be called on objects created in the parent
   ## and thus cause double cleanup.  For instance, if DESTROY() unlinks
   ## a file in the child, we don't want the parent to suddenly miss
   ## it.
   POSIX::exit 0;

there is also documentation talking about POSIX::exit[1].  In fact POSIX::exit() just calls the 
normal perl builtin function exit().  The function you want to call to avoid destructors is 
POSIX::_exit().

[1] http://search.cpan.org/grep?cpanid=ADAMK&release=IPC-Run-
0.82&string=POSIX%3A%3Aexit&i=1&n=1&C=0
```

From toddr@cpan.org on 2010-03-23 04:43:38:
```
I'm helping with the maintenance of this module. I'd feel better if we could 
put a test in  place for this any ideas?
```

**Comment by toddr** [comment]
@gisle for starters, I think you're right.

This looks to be what #86 is trying to address via Pull Request #102. I'm going to treat this ticket as a dupe for now. until I can sort out why #86 is failing.

**Comment by toddr** [comment]
Actually fixed via #41
